flexmeasures/data/scripts/grid_weather.py

```diff
@@ -317,22 +318,27 @@ def get_data_source() -> DataSource:
     return data_source
 
 
-def call_darksky(api_key: str, location: Tuple[float, float]) -> dict:
-    """Make a single call to the Dark Sky API and return the result parsed as dict"""
-    return ForecastIO.ForecastIO(
-        api_key,
-        units=ForecastIO.ForecastIO.UNITS_SI,
-        lang=ForecastIO.ForecastIO.LANG_ENGLISH,
-        latitude=location[0],
-        longitude=location[1],
-        extend="hourly",
-    ).forecast
+def call_openweatherapi(
+    api_key: str, location: Tuple[float, float]
+) -> Tuple[int, List[Dict]]:
+    """
+    Make a single "one-call" to the Open Weather API and return the API timestamp as well as the 48 hourly forecasts.
+    See https://openweathermap.org/api/one-call-api for docs.
+    Note that the first forecast is about the current hour.
+    """
+    query_str = f"lat={location[0]}&lon={location[1]}&units=metric&exclude=minutely,daily,alerts&appid={api_key}"
+    res = requests.get(f"http://api.openweathermap.org/data/2.5/onecall?{query_str}")
```

Comment:
Is this 2.5 their current API version? If so, let's set up a place for version management of external services. Or it could be a config variable.

Reply:
I don't know what the 2.5 really is. they promote it in all documentation. There is no trace of earlier versions anywhere on their site. I'd like to leave this as-is for now.